Tests/Unit/SentryComponentTest.php

```diff
@@ -169,9 +180,12 @@ public function testRegisterRavenWithGuestUserContext(): void
 		/** @var IApplicationComponent&\Mockery\MockInterface $user */
 		$user = \Mockery::mock(IApplicationComponent::class);
 		$user->shouldReceive('getIsInitialized')->andReturnTrue();
-		$user->isGuest = true;
 		$user->shouldReceive('getId')->andReturn($userId)->never();
 		$user->shouldReceive('getName')->andReturn($userName)->never();
+
+		/** @phpstan-ignore-next-line */
+		$user->isGuest = true;
```

Comment:
We'll need to watch for this when moving to PHP 8.2. Dynamic properties are deprecated, and this is a dynamic property on the mock object.